config/rewrite/user.role.authenticated.yml

```diff
@@ -0,0 +1,14 @@
+langcode: en
+status: true
+dependencies: {  }
+id: authenticated
+label: 'Authenticated user'
+weight: -9
+is_admin: false
+permissions:
```

Comment:
@saurabh-2k17 we only need to rewrite permission key, please remove all the above code
 `permissions:
  - 'access content'
  - 'access moderation notes'
  - 'access shortcuts'
  - 'create moderation notes'
  - 'use text format basic_html'
  - 'view search api pages'`